com.github.mdh34.quickdocs.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "com.github.mdh34.quickdocs",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
```

Comment:
Does it work with 3.28?

Reply:
My memory has been that elementary apps like to match the versions of libraries etc. on the elementary platform, so they often have older runtimes (which is fine! we want to make sure things use the tested set). However obviously the newer is the 'best' as we'd like to reduce version skew and bandwidth and storange usage for end users.

Reply:
I've been building this on the juno beta which uses the 3.28 stack so it isn't a problem :)